### PR TITLE
0.17.0 release

### PR DESCRIPTION
**Pull request by reset**
gif-keyboard-5956529056213154129

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by fnichol** [comment]
x0snq

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,118 @@
...
+- revert change to build.ps1 [\#1715](https://github.com/habitat-sh/habitat/pull/1715) ([smurawski](https://github.com/smurawski))
+-  Rename pkg\_expose to pkg\_exposes and refactor to leverage pkg\_exports [\#1712](https://github.com/habitat-sh/habitat/pull/1712) ([reset](https://github.com/reset))
+- Update Building.md for Windows to match our current process [\#1709](https://github.com/habitat-sh/habitat/pull/1709) ([smurawski](https://github.com/smurawski))
+- Depot validates hab client target \(platform/architecture\) before returning packages [\#1707](https://github.com/habitat-sh/habitat/pull/1707) ([smurawski](https://github.com/smurawski))
+- \[hab\] Install same version of helper packages as hab if not present. [\#1700](https://github.com/habitat-sh/habitat/pull/1700) ([fnichol](https://github.com/fnichol))
+- Document the log keys [\#1699](https://github.com/habitat-sh/habitat/pull/1699) ([smith](https://github.com/smith))
+- \[hab-spider\] Remove release profile from Cargo.toml. [\#1696](https://github.com/habitat-sh/habitat/pull/1696) ([fnichol](https://github.com/fnichol))
```
Comment (chefsalim) [inline CHANGELOG.md]: Remove the hab spider changes from the list since it is currently only an internal tool?

Comment (smurawski) [inline CHANGELOG.md]: @chefsalim You'll want to label them that way with github labels, otherwise the next gen of the release will add them back.

**Comment by echohack** [comment]
ncley

**Pull request merged**